Makefile

```diff
@@ -11,22 +11,46 @@ endif
 init:
 	${pip} install .
 
-dev-deps:
-	${pip} install .[test,lint]
+dev-init:
+	${pip} install .[dev]
 
 test:
-	python setup.py test
+	pytest
 
 lint:
 	tox -e lint
 
-clean: clean-build clean-pyc clean-test
+docs:
+	rm -f docs/vyper.rst
+	rm -f docs/modules.rst
+	sphinx-apidoc -o docs/ -d 2 vyper/
+	$(MAKE) -C docs clean
+	$(MAKE) -C docs html
+	# To display docs, run:
+	# open docs/_build/html/index.html (macOS)
+	# start docs/_build/html/index.html (Windows)
+	# xdg-open docs/_build/html/index.html (Linux)
+
+release: clean
+	python setup.py sdist bdist_wheel
+	twine check dist/*
+	#twine upload dist/*
+
+clean: clean-build clean-docs clean-pyc clean-test
 
 clean-build:
 	@echo Cleaning python build files...
 	@rm -fr build/
 	@rm -fr dist/
 	@rm -fr *.egg-info
+	@rm -f vyper/vyper_git_version.txt
+
+clean-docs:
+	@echo Cleaning doc build files...
+	@rm -rf docs/_build/
+	@rm -f docs/modules.rst
+	@rm -f docs/vyper.rst
+	@rm -f docs/vyper.*.rst
```

Comment:
not sure these lines do anything

Reply:
they do, these are intermediate build files for sphinx